reana_server/config.py

```diff
@@ -106,12 +112,19 @@ def _(x):
 )
 
 CERN_APP_CREDENTIALS = dict(
-    consumer_key='CHANGE_ME',
-    consumer_secret='CHANGE_ME',
+    consumer_key=CERN_CONSUMER_KEY,
+    consumer_secret=CERN_CONSUMER_SECRET,
 )
 
 DEBUG = True
 
 SECURITY_PASSWORD_SALT = 'security-password-salt'
 
 SECURITY_SEND_REGISTER_EMAIL = False
+
+# Gitlab Application configuration
+# ================================
+GITLAB_APP_ID = os.getenv('GITLAB_APP_ID', 'CHANGE_ME')
+GITLAB_APP_SECRET = os.getenv('GITLAB_APP_SECRET', 'CHANGE_ME')
+GITLAB_URL = os.getenv('GITLAB_URL', 'CHANGE_ME')
```

Comment:
The `REANA_GITLAB_OAUTH` prefix works for all GitLab variables but this one, this could simply be `REANA_GITLAB_URL`.

Reply:
Since we are using this URL to do a [direct request to GitLab including the application secret](https://github.com/reanahub/reana-server/pull/165/files#diff-eb771db72945ea9e2b49545b0938ce74R93), we should somehow force the HTTPS with something like:
```python
GITLAB_URL = 'https://{}'.format(os.getenv('REANA_GITLAB_HOST', 'CHANGE_ME'))
```